plutus-ledger/src/Ledger/Orphans.hs

```diff
@@ -80,19 +84,6 @@ instance Serialise (C.AddressInEra C.BabbageEra) where
       pure
       $ C.deserialiseFromRawBytes (C.AsAddressInEra C.AsBabbageEra) bs
 
-instance Ord (C.AddressInEra C.BabbageEra) where
-  compare (C.AddressInEra C.ByronAddressInAnyEra addr1)
-       (C.AddressInEra C.ByronAddressInAnyEra addr2) = compare addr1 addr2
-
-  compare (C.AddressInEra C.ShelleyAddressInEra{} addr1)
-       (C.AddressInEra C.ShelleyAddressInEra{} addr2) = compare addr1 addr2
-
-  compare (C.AddressInEra C.ByronAddressInAnyEra _)
-       (C.AddressInEra C.ShelleyAddressInEra{} _) = LT
-
-  compare (C.AddressInEra C.ShelleyAddressInEra{} _)
-       (C.AddressInEra C.ByronAddressInAnyEra _) = GT
-
```

Comment:
It's weird you were able to remove it without making anything break, I thought it would have been use in a few other places.

Reply:
We define the same instance in quickcheck-dynamic so that's why it works.

Reply:
We're getting it from quickcheck-contractmodel instead of defining it here. We need it in that repo so it also had an orphan instance.